devops/charts/server/templates/deployment.yaml

```diff
@@ -39,14 +39,16 @@ spec:
             httpGet:
               path: /health
               port: http
-            initialDelaySeconds: 10
-            periodSeconds: 3
+            initialDelaySeconds: 60
+            timeoutSeconds: 5
+            periodSeconds: 10
           readinessProbe:
             httpGet:
               path: /health
               port: http
-            initialDelaySeconds: 10
-            timeoutSeconds: 3
+            initialDelaySeconds: 60
```

Comment:
This may or may not be desired. The pod does not enter the READY state until the "readyness probe" (if one id specified) succeeds at least once. The service will not route traffic to a pod until it is READY to accept traffic. With `initialDelaySeconds: 60` your pod won't accept any traffic after a restart for a minimum of 60 seconds. This is probably Ok in dev/test but is an unnecessary delay in prod.

Reply:
adjusted